src/clr-core/welcome.stories.ts

```diff
@@ -62,6 +62,8 @@ export const gettingStarted = () => {
 
       <p>
         Clarity Core is a Web Component implementation of the <a href="https://clarity.design">Clarity Design System</a>.
+        Clarity Core provides a set of reusable UI components that work in any JavaScript framework or no framework at all.
+        You can read more about Core and its goals in our <a href="https://medium.com/claritydesignsystem/clarity-core-72f6d3a029bc">Medium blog post</a>.
```

Comment:
TODO: going to add a comment that the initial batch of components we will be treating as Beta but feel they are stable enough for teams to start trying and work with getting feedback on.